### PR TITLE
Restored PUT on hook-status

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Please be specific when describing this change. What did you change? Why?
| Type?             | bug fix 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Sponsor company   | PrestaShop SA
| How to test?      | CI is green

### CODE REVIEW

**Review by jolelievre — DISMISSED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
